src/Akka.Persistence.MongoDb/Query/QueryApi.cs

```diff
@@ -25,6 +23,7 @@ public interface ISubscriptionCommand { }
     /// the subscriber when <see cref="AsyncWriteJournal.WriteMessagesAsync"/> has been called.
     /// </summary>
     [Serializable]
+    [Obsolete("Query is not implemented.", true)]
     public sealed class SubscribePersistenceId : ISubscriptionCommand
```

Comment:
Database-driven versions of these queries, which work properly, still exist though right?

Reply:
That is part of `Persistence.Query` which is different than these

Reply:
Yes, this PR would close #346 and #345, its a simple fix, we just haven't done it yet